src/components/sync-connected-sites.tsx

```diff
@@ -193,27 +197,36 @@ export function SyncConnectedSites( {
 											</div>
 										) }
 										{ ! isPulling && ! hasPullFinished && ! isError && (
-											<div className="flex gap-2 pl-4 ml-auto shrink-0 h-5">
-												<Button
-													variant="link"
-													className="!text-black hover:!text-a8c-blueberry"
-													onClick={ () => {
-														pullSite( connectedSite, selectedSite );
-													} }
-													disabled={ isAnySitePulling }
-												>
-													<Icon icon={ cloudDownload } />
-													{ __( 'Pull' ) }
-												</Button>
-												<Button
-													variant="link"
-													className="!text-black hover:!text-a8c-blueberry"
-													disabled={ isAnySitePulling }
-												>
-													<Icon icon={ cloudUpload } />
-													{ __( 'Push' ) }
-												</Button>
-											</div>
+											<Tooltip
```

Comment:
I added one tooltip for these two buttons as they are too close too each other and are too small and making two tooltips seemed too crowded and excessive. Happy to change this though

Reply:
That makes sense. Good call @katinthehatsite